src/libraries/System.Net.Sockets/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -579,8 +581,15 @@ public void SendPacketsElement_FileStreamLargeCount_Throws()
             }
         }
 
-        [Fact]
+        [ConditionalFact]
         public void SendPacketsElement_FileStreamMultiPartMixed_MultipleFileStreams_Success() {
+            
+            if (PlatformDetection.IsWindows10Version22000OrGreater)
+            {
+                // [ActiveIssue("https://github.com/dotnet/runtime/issues/58898")]
+                throw new SkipTestException("Unstable on Windows 11");
+            }
```

Comment:
I'm seeing repeated failures in multiple SendPacketsElement tests, not just this one, e.g.
![image](https://user-images.githubusercontent.com/2642209/134017781-21c119ad-4c99-4ec3-8c1f-fb232c8f3083.png)
Can we get them all disabled asap?
cc: @karelz